cosmic_ray/operators/remove_decorator.py

```diff
@@ -0,0 +1,27 @@
+import ast
+
+from .operator import Operator
+
+
+class RemoveDecorator(Operator):
+    """An operator that removes each of the non standard decorators."""
+    regular_decorators = ["classmethod", "staticmethod", "abstractmethod"]
+
+    def visit_FunctionDef(self, node):  # noqa
+        decorator_candidates = [x for x in node.decorator_list if \
+                                x.id not in self.regular_decorators]
+        if decorator_candidates:
+            return self.visit_mutation_site(node, len(decorator_candidates))
+
+        return node
+
+    def mutate(self, node, idx):
+        """Modify the decorator list to remove one decorator at each mutation"""
+        decorator_candidates = [x for x in node.decorator_list if \
+                                x.id not in self.regular_decorators]
+        removed_decorator = decorator_candidates[idx]
+        modified_decorator = [x for x in node.decorator_list if \
+                              x.id != removed_decorator.id]
```

Comment:
agreed, at the beginning I thought that no one would put more than one decorator with the same name, but it might be the case especially if is parametrized